sources/ToggleRouter.php

```diff
@@ -97,4 +97,28 @@ public function configureFeature(string $feature, string $strategy, string $meth
             'parameters' => $parameters,
         ]);
     }
+
+    public function listUnregisteredFeatures(): array
+    {
+        $configuredFeatures = array_unique(array_reduce(
+            $this->strategies,
+            fn (array $features, TogglingStrategy $strategy) => \array_merge($features, $strategy->listFeatures()),
+            []
+        ));
+
+        $registeredFeatures = array_keys($this->registry->getDefinitions());
+
+        $unregisteredFeatures = array_values(array_diff($configuredFeatures, $registeredFeatures));
+
+        sort($unregisteredFeatures);
```

Comment:
Not sure if this should be part of this API or in the clients of this API: it feels like sorting things should be dealt with at display time...